aaa.svelte

```diff
@@ -0,0 +1,8 @@
+<script>
+  var name = 'aaaaaaaaaaaaaaaaa';
```

Comment:
Could you make this add only one `a` at a time, so it's consistent with the rest of the files? See [aaa.html](aaa.html)

Reply:
oh oops yeah